src/common/datetime.c

```diff
@@ -0,0 +1,129 @@
+/*
+    This file is part of darktable,
+    Copyright (C) 2011-2022 darktable developers.
```

Comment:
I meant 2022 only. This is a new unit introduce now.

Reply:
Understood, sorry. Thanks.